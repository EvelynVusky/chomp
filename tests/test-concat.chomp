int main() {
	bit a = {{1}};
    nibble b = {{0101}};
    byte c = {{11100010}};
    word d = {{0111000110011110}};

    nibble b_new = a >< a;
    println(b_new);
    byte c_new = a >< b;
    println(c_new);
    word d_new = a >< c;
    println(d_new);
    int i_new = a >< d;
    println(i_new);

    c_new = b >< b;
    println(c_new);
    d_new = b >< c;
    println(d_new);
    i_new = b >< d;
    println(i_new);

    d_new = c >< c;
    println(d_new);
    i_new = c >< d;
    println(i_new);

	return 0;
}