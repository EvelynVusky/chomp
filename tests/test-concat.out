0011
00010101
0000000111100010
94622
01010101
0000010111100010
356766
1110001011100010
14840222
